easybuild/easyconfigs/k/KronaTools/KronaTools-2.8.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,41 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+# this is a bug fix to make sure the symlinks in bin are not getting destroyed 
+# by Easybuild when it is tidying up
+# This build also links updateTaxonomy.sh and updateAccessions.sh in the bin folder
+# so users can install their own Taxonomy database
+
+easyblock = 'Tarball'
+
+name = 'KronaTools'
+version = '2.8.1'
+
+homepage = 'https://github.com/marbl/Krona/wiki/KronaTools'
+description = """Krona Tools is a set of scripts to create Krona charts from 
+several Bioinformatics tools as well as from text and XML files."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
```

Comment:
if these are just bash and perl scripts, there doesn't seem to be a reason to use `GCC` instead of `GCCcore`, is there?

Reply:
Thank you for the hint, I will change this.